packages/api/controllers/nomination.js

```diff
@@ -304,14 +290,35 @@ const getAwaitingHipaa = async () => {
         nomination.hospitalState,
       );
       const applicationName = `${lastName}-${state}`;
-
-      applicationsAwait.push(applicationName);
+      applicationsAwait[applicationName] = nomination.id;
     });
   }
-
   return applicationsAwait;
 };
-
+const getVerifiedNoms = async () => {
+  const nominations = await db.Nomination.findAll({ where: { status: { [Op.or]: ['HIPAA Verified', 'Document Review', 'Ready for Board Review'] } } });
+  const applicationsAwait = {};
+  if (nominations.length) {
+    nominations.forEach((nomination) => {
+      const lastName = nomination.patientName
+        ? nomination.patientName.split(' ')[1]
+        : '';
+      const state = states.getStateCodeByStateName(
+        nomination.hospitalState,
+      );
+      const applicationName = `${lastName}-${state}`;
+      applicationsAwait[applicationName] = nomination.id;
+    });
+  }
+  return applicationsAwait;
+};
+const updateDashboard = async (trueFalse, nominationId) => {
```

Comment:
Can we think of a more descriptive variable name than `trueFalse`?

Reply:
changed